doc/releases/release-notes-2.1.rst

```diff
@@ -141,4 +660,305 @@ release:
+* :github:`18488` - Bluetooth: Mesh: Friend queue message seqnum order
+* :github:`18480` - Microchip's MEC1501 HAL is broken (watchdog part)
+* :github:`18465` - timeutil_timegm() has undefined behavior
+* :github:`18440` - [Coverity CID :203439]Memory - corruptions in /tests/kernel/mem_protect/protection/src/main.c
+* :github:`18373` - [Coverity CID :203399]API usage errors in /samples/boards/olimex_stm32_e407/ccm/src/main.c
+* :github:`18340` - settings: make NVS the default backend
+* :github:`18308` - net: TCP/IPv6 set of fragmented packets causes Zephyr to quit
+* :github:`18305` - Native Posix target can not use features with newlib dependencies
+* :github:`18297` - Bluetooth: SMP: Pairing issues
+* :github:`18282` - tests/kernel/sched/schedule_api/ fails on LPC54114_m4
+* :github:`18160` - Cleanup dts compatible for "nxp,kinetis-sim" on nxp_ke1xf
+* :github:`18143` - stm32f SPI Slave TX does not work correctly, but occurs OVERRUN err
+* :github:`18138` - xtensa arch has two different implementations
+* :github:`18105` - BSD socket offload with IPv4 and IPv6 disabled breaks many client-based net samples
+* :github:`17998` - STM32 (Nucleo L476RG) SPI pins floating
+* :github:`17983` - Bluetooth: Re-establish security before notifications/indications can be sent
+* :github:`17949` - stm32 i2c driver has problems with AHB_PRESCALER, APB1_PRESCALER, APB2_PRESCALER
+* :github:`17892` - arch/x86: clean up segmentation.h
+* :github:`17888` - arch/x86: remove IAMCU ABI support
+* :github:`17832` - x86: update mmustructs.h and x86_mmu.c to support long mode
+* :github:`17829` - support default property values in devicetree bindings
+* :github:`17781` - Question:Is it possible to connect the device on internet using bluetooth connection?
+* :github:`17625` - driver: gpio: PCAL9535A: can't write to register (read is possible)
+* :github:`17548` - Can't set thread name with k_thread_create prevents useful tracing information
+* :github:`17546` - Bluetooth: Central Scan fails continuously if last connect attempt failed to complete
+* :github:`17443` - Kconfig: move arch-specific stack sizes to arch trees?
+* :github:`17430` - arch/x86: drivers/interrupt_controller/system_apic.c improperly classifies IRQs
+* :github:`17361` - _THREAD_QUEUED overlaps with x86 _EXC_ACTIVE in k_thread.thread_state
+* :github:`17337` - ArmV7-M mpu sub region alignment
+* :github:`17239` - Too many open files crash when running "sanitycheck" with no arguments
+* :github:`17234` - CONFIG_KERNEL_ENTRY appears to be superfluous
+* :github:`17133` - arch/x86: x2APIC EOI should be inline
+* :github:`17104` - arch/x86: fix -march flag for Apollo Lake
+* :github:`17064` - drivers/serial/uart_ns16550: CMD_SET_DLF should be removed
+* :github:`16900` - Inline assembly in Arm z_arch_switch_to_main_thread missing clobber list
+* :github:`16880` - Systematic *-zephyr-eabi/bin/ld: warning: toolchain_is_ok cannot find entry symbol _start; defaulting to 000::00XXXXX
```

Comment:
The script that creates these lists of issues needs to scan for meaningful reST syntax and escape those so they're not interpreted by Sphinx.  In this case replace ``*`` with ``\*``

Reply:
Okay. Will do.